### ISSUE TITLE
feed.php errors

### ISSUE THREAD

**Issue by Miro-Collas** [dokuwiki/dokuwiki#4030]
PHP 8.1
Dokuwiki Release 2023-04-04a "Jack Jackrum"

logs spammed with a lot of these:

```
2023-08-18 19:12:32E_WARNING: Undefined array key "media"/home/fswiki/public_html/feed.php(385)
    #0 /home/fswiki/public_html/feed.php(385): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/fswiki/pu...', 385)
    #1 /home/fswiki/public_html/feed.php(96): rss_buildItems(Object(UniversalFeedCreator), Array, Array)
    #2 {main}

2023-08-18 19:12:32E_WARNING: Undefined array key "media"/home/fswiki/public_html/feed.php(235)
    #0 /home/fswiki/public_html/feed.php(235): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/fswiki/pu...', 235)
    #1 /home/fswiki/public_html/feed.php(96): rss_buildItems(Object(UniversalFeedCreator), Array, Array)
    #2 {main}

2023-08-18 19:12:32E_WARNING: Undefined array key "media"/home/fswiki/public_html/feed.php(290)
    #0 /home/fswiki/public_html/feed.php(290): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/fswiki/pu...', 290)
    #1 /home/fswiki/public_html/feed.php(96): rss_buildItems(Object(UniversalFeedCreator), Array, Array)
    #2 {main}

2023-08-18 19:12:32E_WARNING: Undefined array key "media"/home/fswiki/public_html/feed.php(385)
    #0 /home/fswiki/public_html/feed.php(385): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/fswiki/pu...', 385)
    #1 /home/fswiki/public_html/feed.php(96): rss_buildItems(Object(UniversalFeedCreator), Array, Array)
    #2 {main}
```

**Comment by Miro-Collas** [comment]
there is more which may be related:

```
2023-08-18 21:49:20E_WARNING: Undefined array key "media"/home/fswiki/public_html/feed.php(385)
    #0 /home/fswiki/public_html/feed.php(385): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/fswiki/pu...', 385)
    #1 /home/fswiki/public_html/feed.php(96): rss_buildItems(Object(UniversalFeedCreator), Array, Array)
    #2 {main}
2023-08-18 21:49:23E_ERROR: Maximum execution time of 45 seconds exceeded/home/fswiki/public_html/inc/Parsing/Lexer/ParallelRegex.php(105)
    #0 [internal function]: dokuwiki\ErrorHandler::fatalShutdown()
    #1 {main}
```

**Comment by fiwswe** [comment]
I'm not sure the E_ERROR has anything to do with the E_WARNING. There is not enough context to figure out what triggered it.

The E_WARNING is a duplicate of what you already reported. And it should be fixed by #4031.

**Comment by dregad** [comment]
Duplicate of #3970
